src_c/event.c

```diff
@@ -1667,9 +1667,18 @@ pg_event_post(PyObject *self, PyObject *args)
         Py_RETURN_NONE;
     }
 
-    if (pgEvent_FillUserEvent(e, &event))
-        return NULL;
+    if (e->type == SDL_KEYDOWN || e->type == SDL_KEYUP){
+        event.type =  e->type;
+        event.key.keysym.sym = PyLong_AsLong(PyDict_GetItemString(e->dict, "key"));
+        if (PyDict_GetItemString(e->dict, "scancode") != NULL)
+            event.key.keysym.scancode = PyLong_AsLong(PyDict_GetItemString(e->dict, "scancode"));
```

Comment:
You may want to also add the `window` field for `SDL_KEYDOWN `and `SDL_KEYUP `events
and the `unicode `field for `SDL_KEYDOWN `events.

Reply:
unicode is not a SDL2 thing, but a pygame thing!
https://wiki.libsdl.org/SDL_Keysym
https://wiki.libsdl.org/SDL_KeyboardEvent